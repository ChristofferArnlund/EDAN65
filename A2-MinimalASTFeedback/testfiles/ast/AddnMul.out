Program
  List
    Func
      IdDecl ID="a"
        Opt
      List
      StmtBlock
        List
          IdDecl ID="x"
            Opt
              Numeral NUMERAL="0"
          IdDecl ID="y"
            Opt
          Assign
            IdUse ID="x"
            Add
              Mul
                Numeral NUMERAL="5"
                Numeral NUMERAL="2"
              Numeral NUMERAL="3"
          Assign
            IdUse ID="y"
            Add
              Numeral NUMERAL="3"
              Mul
                Numeral NUMERAL="5"
                Numeral NUMERAL="2"
