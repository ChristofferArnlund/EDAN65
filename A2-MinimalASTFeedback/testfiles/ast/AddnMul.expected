Program
  List
    Func
      IdDecl
        Name ID="a"
        Opt
      List
      StmtBlock
        List
          IdDecl
            Name ID="x"
            Opt
              Numeral NUMERAL="0"
          IdDecl
            Name ID="y"
            Opt
          Assign
            IdUse
              Name ID="x"
            Add
              Mul
                Numeral NUMERAL="5"
                Numeral NUMERAL="2"
              Numeral NUMERAL="3"
          Assign
            IdUse
              Name ID="y"
            Add
              Numeral NUMERAL="3"
              Mul
                Numeral NUMERAL="5"
                Numeral NUMERAL="2"
